src/spetlr/delta/delta_handle.py

```diff
@@ -80,13 +80,13 @@ def __init__(
     def from_tc(cls, id: str) -> "DeltaHandle":
         tc = Configurator()
         return cls(
-            name=tc.table_property(id, "name", ""),
-            location=tc.table_property(id, "path", ""),
+            name=tc.get(id, "name", ""),
```

Comment:
Perhaps a default setting of `None` or `""` in the `.get`-method for the last argument would remove some boilerplate?

Reply:
I disagree! The default argument explicitly allows you to provide None or "" as valid arguments if you do not want the method to throw an exception in case the key does not exist. Analogous to default dict.get() behavior.